pkg/api/v1beta1/dynakube/validation/oneagent.go

```diff
@@ -17,14 +18,18 @@ const (
 `
 	errorImageFieldSetWithoutCSIFlag = `The DynaKube's specification tries to enable ApplicationMonitoring mode and get the respective image, but the CSI driver is not enabled.`
 
-	errorNodeSelectorConflict = `The DynaKube's specification tries to specify a nodeSelector conflicts with an another Dynakube's nodeSelector, which is not supported.
-The conflicting Dynakube: %s
-`
+	errorNodeSelectorConflict = `The DynaKube specification attempts to deploy a %s, which conflicts with another DynaKube's %s. Only one Agent per node is supported.
+Conflicting DynaKube: %s`
```

Comment:
can more than one Dynakube be conflicting? Example: 2 Dynakubes with node selector and I try to apply one without a node selector

Reply:
The error will state the first collision detected, just as we did only for the OneAgent before

Reply:
I know, it's more confusing with oneagent/logmodule both existing now and only the first of them included in the warning

Reply:
So what do you suggest?

Reply:
couldn't just have something like:

```
var conflicts []string
for _, item := range validDynakubes.Items {
...

	if hasConflictingMatchLabels(oneAgentNodeSelector, item.OneAgentNodeSelector()) {
		...
		conflicts = append(conflicts, fmt.Sprintf(errorNodeSelectorConflict, oneAgentComponentName, oneAgentComponentName, item.Name))
	}
...
}

if len(conflicts)>0 {
	...	// reject
}
```

be careful not to just spit out a wall of text though 😅

Reply:
Will try to update it tomorrow to consider multiple dynakubes and features 👍